src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -384,6 +384,39 @@
               --connector-name aci-connector --chart-url <CustomURL> --image-tag <VirtualKubeletImageTag>
 """
 
+helps['aks use-dev-spaces'] = """
+    type: command
+    short-summary: (PREVIEW) Use Azure Dev Spaces with a managed Kubernetes cluster.
+    parameters:
+        - name: --name -n
```

Comment:
I don't see any other place in this file where the -n, -g options are explicitly documented. I think they get inherited from some parent context. What happens if you remove them? The main reason I ask is that the short-summary values you provide are lacking newlines in appropriate places to make sure the lines don't get too long. I think by letting something else include these, the wrapping will be done properly.

Reply:
Look at _params.py. There's some "aks" context that defines the -g and -n options so you don't have to.